project-koku/koku
# [Cost-2031] - Adding savingsplan to postgres

#### Adding Saving plan to postgres processes:

#### Testing Instructions:
1. Pull and Checkout branch
2. Bring up Docker 
3. Create and Load test costumer Data
4. hit the following Endpoints:
  For the following tests the values for blended and unblended should match as they do in the nise data. Savings plan should be the only cost type which differs in value.
```
Endpoint for unblended_cost:
http://127.0.0.1:8000/api/cost-management/v1/reports/aws/costs/?filter%5Btime_scope_value%5D=-1&check_tags=True

Endpoint for blended_cost:
http://127.0.0.1:8000/api/cost-management/v1/reports/aws/costs/?filter%5Btime_scope_value%5D=-1&check_tags=True&cost_type=blended_cost

Endpoint for savingsplan:
http://127.0.0.1:8000/api/cost-management/v1/reports/aws/costs/?filter%5Btime_scope_value%5D=-1&check_tags=True&cost_type=savingsplan_effective_cost

```
Repeat steps 2-4 for both bringing docker up in Trino and postgres:
